contracts/security/src/events/update_security_types.rs

```diff
@@ -0,0 +1,60 @@
+use cosmwasm_std::Event;
+
+use crate::core::constants::UPDATE_SECURITY_TYPES_EVENT;
+
+/// An event to represent the contract's supported security types changing.
+pub struct UpdateSecurityTypesEvent {}
+
+impl UpdateSecurityTypesEvent {
+    /// Creates a new instance of UpdateSecurityTypesEvent
+    ///
+    /// # Arguments
+    ///
+    /// # Examples
+    /// ```
+    /// let event = UpdateSecurityTypesEvent::new();
+    /// ```
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+/// Allows us to easily convert a UpdateSecurityTypesEvent into an Event.
+impl From<UpdateSecurityTypesEvent> for Event {
+    fn from(_val: UpdateSecurityTypesEvent) -> Self {
```

Comment:
This event seems just to emit what is happening, but no further detail.  Do other events get event with more detail about the security being updated?

Reply:
The two messages AddSecurityTypes and RemoveSecurityTypes both use the UpdateSecurityTypesEvent. I was hesitant to add the types to the event in case we need to add or remove a large number of types at once. I thought it would be safer to at least notify any listeners that there was an update, and they could always query for the types. If you think it's better to have the types in the event, then I can add them to both add and remove.

Reply:
Yes, that is perfect!